package.json

```diff
@@ -39,6 +40,13 @@
   "devDependencies": {
     "browserify": "^17.0.0",
     "budo": "^11.8.4",
-    "esmify": "^2.1.1"
+    "esmify": "^2.1.1",
+    "prettier": "2.8.0"
+  },
+  "engines": {
+    "node": ">=14 <18"
+  },
+  "volta": {
+    "node": "17.9.1"
   }
```

Comment:
[Volta ](https://docs.volta.sh/guide/) is super helpful for managing JS command line tools. It sets the Node CLI to use the one specified in the package.json file. It helps developers who use multiple Node versions (like I do). Highly recommend trying it out.

Totally fine to not use it - it is ignored by npm. So having this addition here is harmless.